docs/data/material/getting-started/templates/templates.md

```diff
@@ -28,4 +29,6 @@ Please feel free to open an [issue](https://github.com/mui/material-ui/issues/ne
 
 Looking for something more? You can find complete templates and themes in the <a href="https://mui.com/store/?utm_source=docs&utm_medium=referral&utm_campaign=templates-store">premium template section</a>.
 
-<a href="https://mui.com/store/?utm_source=docs&utm_medium=referral&utm_campaign=templates-store"><img src="/static/images/themes-light.jpg" alt="react templates" width="2278" height="1358" /></a>
+<a href="https://mui.com/store/?utm_source=docs&utm_medium=referral&utm_campaign=templates-store">
+<img src="/static/images/themes-display.png" alt="react templates" width="1140" height="600" />
```

Comment:
As a side note, it might be best to use the actual image dimension, 

```suggestion
<a href="https://mui.com/store/?utm_source=docs&utm_medium=referral&utm_campaign=templates-store">
<img src="/static/images/themes-display.png" alt="react templates" width="2280" height="1200" />
```

With @Janpot we explored improving the image infrastructure https://www.notion.so/mui-org/Docs-infra-Images-solution-30e78dbf24744ed5b570027656139092, it seems that its what we will need DOM to contain, but we will add automation to save time. For example what Next.js has in its DOM: https://nextjs.org/docs/app/building-your-application/data-fetching/patterns#streaming.

I have updated https://www.notion.so/mui-org/High-quality-media-5a1686758f4f4267989b67a1c2c270da?pvs=4#2c953181fe534200ac688d414ca0bf85 to reflect this.